terraform/s3.tf.json.template.py

```diff
@@ -23,6 +23,15 @@
                 'force_destroy': True
             }
         },
+        'aws_s3_bucket_public_access_block': {
+            'storage': {
```

Comment:
Even though there is currently only one bucket in a deployment, this should use the same `for_each` technique as in the `shared` component.

Reply:
Addressed during PL.